calci.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
add css library for more it visually  appealing

Reply:
Done